src/styles/about-us.scss

```diff
@@ -0,0 +1,41 @@
+.about-us {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 104px;
+  color: $darkColor;
+  @media (max-width: $displayLarge) {
+    flex-direction: column;
+    padding-top: 85px;
+  }
+  @media (max-width: $displaySmall) {
+    padding-top: 64px;
+  }
+
+  &__top-title {
+    font-size: $title2LvlFontSize;
```

Comment:
too abstract name, what is 2 level, how big is comparing to others and other questions are coming up when I see this name

Reply:
ok

Reply:
not fixed

Reply:
ok, what name do you propose?

Reply:
for title: primary title, secondary title and so on

Reply:
ok